core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -1506,7 +1508,7 @@ class ReplicaManagerTest {
       purgatoryName = "ElectLeader", timer, reaperEnabled = false)
 
     // Mock network client to show leader offset of 5
-    val quota = QuotaFactory.instantiate(config, metrics, time, "")
+    val quota = quotaManager
```

Comment:
nit: Could we get rid of `quota`?

Reply:
this is needed to avoid method variable name collision with the `ReplicationQuotaManager` in the overriden `createReplicaFetcherManager` a little bit further down in the test.

Reply:
Could we rename `quotaManager` used in `createReplicaFetcherManager` to `replicationQuotaManager` and use `quotaManager` as the global one everywhere?